samcli/cli/context.py

```diff
@@ -54,6 +60,22 @@ def debug(self, value):
             logging.getLogger("samcli").setLevel(logging.DEBUG)
             logging.getLogger("aws_lambda_builders").setLevel(logging.DEBUG)
 
+    @output_timestamp.setter
+    def output_timestamp(self, value):
+        """
+        Turn on timestamp logging to print a timestamp at the beginning of each logg entry.
+
+        :param value: Value of output-timestamp flag
+        """
+        self._output_timestamp = value
+
+        for handler in logging.getLogger("samcli").handlers + logging.getLogger("aws_lambda_builders").handlers:
```

Comment:
Since we are using logging.getLogger("samcli") and logging.getLogger("aws_lambda_builders") in quite a few places, taking "samcli" and "aws_lambda_builders" out as constants would be more maintainable.

Reply:
done! good suggestion!